### DISCUSSION TITLE
New User Checklist ✅

### DISCUSSION THREAD

**Discussion by queenofcorgis1** [community/community#27367] [New to GitHub]
Learning GitHub can feel overwhelming especially when deciding where to begin. Luckily, we've compiled a checklist to get you on the track to being a GitHub expert. Complete our checklist below (linked with helpful resources too!) to start your journey to expert status. 

* Create a repository
    * Write your readme for your repo (Bonus points if you upload an image)
    * [Some tips](https://gist.github.com/akashnimare/7b065c12d9750578de8e705fb4771d2f)
* Create a folder in the repo
   * [Click here for help](https://github.community/t/add-a-folder/2304)
* Clone the folder in the repo
   * [How to clone a repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
   * [How to upload an entire folder](https://github.community/t/how-to-upload-an-entire-folder/1056)
* Practice writing in Markdown
   * [Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#videos)
   * Hint: make sure the file has a .md extension (you can add it yourself)
* Add an Issue Template
   * Find Issue Templates [here](https://github.com/devspace/awesome-github-templates)
* Watch a repository you're interested in
   * Some ones we’re watching:
      * [GitHub Training Kit](https://github.com/github/training-kit)
      * [How to Cook](https://github.com/Anduin2017/HowToCook)
      * [Basic Computer Games](https://github.com/coding-horror/basic-computer-games)
* Submit a Pull Request
   * [Resource](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

:white_check_mark: Let us know what you'd like to learn next once you have the basics down.

**Comment by MrSarno** [comment]
This seems like a great idea. :)

I'd suggest re-envisaging this as a repository where people either looking for help or wanting to provide feedback can open issues, people with suggestions can submit pull requests, and there's more formatting options available through using README.md.

**Reply by queenofcorgis1** [reply]
Thanks for the feedback - appreciate it!

**Comment by radar-blip** [comment]
Nice work. 👍🏼 I'm sure it will help some people get on the right path.

**Reply by queenofcorgis1** [reply]
Glad you like it - anything else you think a GitHub user should know/learn how to do?

**Comment by DevZank** [comment]
Thank you very much, this helps me a lot, as I am starting my career as a developer
